tests/api/PDFDocument.spec.ts

```diff
@@ -10,6 +10,7 @@ const unencryptedPdfBytes = fs.readFileSync('assets/pdfs/normal.pdf');
 const oldEncryptedPdfBytes1 = fs.readFileSync('assets/pdfs/encrypted_old.pdf');
 const oldEncryptedPdfBytes2 = fs.readFileSync('pdf_specification.pdf');
 const newEncryptedPdfBytes = fs.readFileSync('assets/pdfs/encrypted_new.pdf');
+const invalidPdfBytes = fs.readFileSync('assets/pdfs/invalid.pdf');
```

Comment:
Can this PDF file be renamed from `assets/pdfs/invalid.pdf` to `assets/pdfs/with_invalid_objects.pdf`? 

Also, I'd like to replace the contents of the file with this one ([with_invalid_objects.pdf](https://github.com/Hopding/pdf-lib/files/3965502/with_invalid_objects.pdf)). The reason being that the file you're currently using is invalid for a number of reasons, not just because it has invalid objects. But [with_invalid_objects.pdf](https://github.com/Hopding/pdf-lib/files/3965502/with_invalid_objects.pdf) is only invalid because of invalid objects. So using it will keep the tests a bit more focused.

Reply:
changed the pdf file, I just tried to build a fake pdf from one of your specs for PDFParser because I don't fully understand the basis for what makes an object invalid in a pdf or not.